pkgs/misc/doge/default.nix

```diff
@@ -0,0 +1,21 @@
+with import <nixpkgs> {};
+with pkgs.python3Packages;
+
+buildPythonPackage rec {
+  pname = "doge";
+  version = "3.5.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256:0lwdl06lbpnaqqjk8ap9dsags3bzma30z17v0zc7spng1gz8m6xj";
```

Comment:
`sha256:` can be removed from the string

Reply:
done